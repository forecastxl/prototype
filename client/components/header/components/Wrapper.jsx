import muiThemeable from 'material-ui/styles/muiThemeable'
import styled from 'styled-components'

export const Wrapper = styled.header`
  background: ${props => props.muiTheme.palette.primary1Color};
  padding: 0.5rem 1rem;
  align-items: center;
  display: flex;
  justify-content: space-between;
`

export default muiThemeable()(Wrapper)
