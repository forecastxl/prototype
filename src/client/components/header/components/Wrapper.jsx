import muiThemeable from 'material-ui/styles/muiThemeable'
import styled from 'styled-components'

export const DumbWrapper = styled.header`
  align-items: center;
  background: ${props => props.muiTheme.palette.primary1Color};
  display: flex;
  justify-content: space-between;
  margin-bottom: 1rem;
  padding: 0.5rem 1rem;
`

export default muiThemeable()(DumbWrapper)
