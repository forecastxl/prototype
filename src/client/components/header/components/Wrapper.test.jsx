/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import { DumbWrapper } from './Wrapper'

describe('<Wrapper />', () => {
  it('renders correctly', () => {
    const theme = {
      palette: {
        alternateTextColor: 'blue'
      }
    }
    const wrapper = shallow(<DumbWrapper muiTheme={theme} />)
    expect(shallowToJson(wrapper)).toMatchStyledComponentsSnapshot()
  })
})
