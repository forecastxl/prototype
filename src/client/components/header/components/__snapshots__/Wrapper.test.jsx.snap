// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> renders correctly 1`] = `
.kHFwzF {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: ;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-between;
  -webkit-justify-content: space-between;
  -ms-flex-pack: space-between;
  justify-content: space-between;
  margin-bottom: 1rem;
  padding: 0.5rem 1rem;
}

<header
  className="sc-bdVaJa kHFwzF"
/>
`;
